content/en/docs/concepts/cluster-administration/system-traces.md

```diff
@@ -47,6 +50,44 @@ service:
       exporters: [logging]
 ```
 
+To directly emit traces to a backend without utilizing a collector, 
+specify the endpoint field in the Kubernetes TracingConfiguration with the desired trace backend address. 
+This method negates the need for a collector and simplifies the overall structure.
+
+For trace backend header configuration, including authentication details, environment variables can be used with `OTEL_EXPORTER_OTLP_HEADERS`, 
+see [OTLP Exporter Configuration](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/).
+
+Additionally, for trace resource attribute configuration such as Kubernetes cluster name, namespace, Pod name, etc., 
+environment variables can also be used with `OTEL_RESOURCE_ATTRIBUTES`, see [OTLP Kubernetes Resource](https://opentelemetry.io/docs/specs/semconv/resource/k8s/).
+
+Below is an example that shows how to configure  to emit traces directly to a backend.
```

Comment:
I'm not sure this example is necessary.  K8s is deployed in a wide variety of ways, and people probably won't have access to a deployment yaml for k8s apiservers.

Reply:
Sure, good comment, I've removed the example as advised. 
Could you please take another look and let me know if there are any additional comments?